build.gradle

```diff
@@ -53,9 +56,14 @@ subprojects {
             dexOptions {
                 preDexLibraries !project.hasProperty('disablePreDex')
             }
+            kotlinOptions {
+                jvmTarget = JavaVersion.VERSION_1_8.toString()
+            }
         }
 
         dependencies {
+            implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:${deps.kotlin}"
```

Comment:
If the java version is 8, why not use [org.jetbrains.kotlin:kotlin-stdlib-jdk8](https://mvnrepository.com/artifact/org.jetbrains.kotlin/kotlin-stdlib-jdk8)

Reply:
good catch, copy and pasted from another project :)

Reply:
although, the only thing that `stdlib-jdk8` brings in is extension functions for java 8 APIs (Streams, Optional, etc). But those APIs can't be used in Android development yet due to backwards compatibility.

Reply:
here's an article about what the various versions of stdlib include related to Android: https://medium.com/@mbonnin/the-different-kotlin-stdlibs-explained-83d7c6bf293

Reply:
Interesting. It sounds like his suggestion then is to just use the base, and not even jdk7.

Reply:
yeah, I'm going to bump it back to the base one. Android is a weird mix of Java 8 due to backwards compatibility